tests/pyopnsense/test_helpers.py

```
@@ -76,35 +75,63 @@ def test_timestamp_to_datetime() -> None:
-    assert pyopnsense_helpers.try_to_float(None, 3.3) == 3.3
-
-
-def test_coerce_bool() -> None:
+@pytest.mark.parametrize(
+    ("value", "default", "expected"),
+    [
+        pytest.param("5", 0, 5, id="numeric-string"),
+        pytest.param(None, 7, 7, id="none-default"),
+    ],
+)
+def test_try_to_int(value: Any, default: int, expected: int) -> None:
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)